### DISCUSSION TITLE
FYI: Incompatible with DimensionalDoors

### DISCUSSION THREAD

**Discussion by No3371** [FxMorin/MoreCulling#152] [General]
If DimensionalDoors is not unchecked in ModCompatibilities config tab, it will results in X-Ray.
![](https://user-images.githubusercontent.com/1814971/226302706-a2c84938-b385-4a9c-a40c-4a4956bcc6c3.png)

**Comment by FxMorin** [comment]
What block specifically?

**Reply by No3371** [reply]
The rift block. Can be created by placing any of the dimensional doors then break it. (Creative Mode)

**Comment by No3371** [comment]
Different mod but I decided to add this here.
![圖片](https://user-images.githubusercontent.com/1814971/228151160-85ca2060-d9ce-467b-8fbe-68bdfea79364.png)
Mod: Adorn
Block: Coffee Table

**Reply by FxMorin** [reply]
All of Adorn models are broken. However I though v1.8 would have fixed that?
What version are you using?

**Reply by No3371** [reply]
Well I tested every block (not every variation though), coffee table is the only one broken. `Adorn-3.8.2+1.19.2-fabric.jar` + `moreculling-1.19.1-0.17.0.jar`

**Comment by No3371** [comment]
![圖片](https://user-images.githubusercontent.com/1814971/228614598-4f410717-cfee-41f0-b04e-60a5b776b073.png)
Apologize, I missed some blocks in the previous Adorn test. This one is the drawer block.
